site/_posts/2017-12-05-last-week-in-manageiq-fixing-bugs.html.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Last week in ManageIQ - Fixing bugs
+author: zita
+date: 2017-12-06
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+Hi. I, [ZitaNemeckova](https://github.com/ZitaNemeckova) will tell you more about our effort to improve our codebase. It's not only about fixing code but also about cleaning and deleting. If you never used static analysis tool to find dead methods this post is for you.
+
+## Deleted
+
+Let's take a small detour and talk about one of our community activities for a bit. Did you know that part of Brno team is collaborating with Mendel University in Brno on teaching students how to contribute to Open Source? 
+
+Probably not as it's in prototyping phase. Last year they helped with conversion of forms in Angular and wrote some tests in Ruby. This year they started our favorite activity: deleting code. 
```

Comment:
Maybe `Last year they helped` to `Last year the students helped` if it sounds better 😉